index.html

```diff
@@ -5,12 +5,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/4e655bdddb.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="./style.css">
     <title>Portfolio</title>
 </head>
 
 <body>
-    <h1>Portfolio Project</h1>
+    <header>
+        <h1 class="logo"><a href="#">Usman</a></h1>
+        <nav>
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
+                    fill="#6070FF" />
+            </svg>
+        </nav>
+
+    </header>
+    <section class="headline-main">
+        <section class="headline">
+            <h1 class="headline-heading">I'm Usman <br> Glad to see you!</h1>
+            <p class="headline-paragraph">I’m a software developer! I can help you build a product , feature or website
+                Look
+                through some of my work
+                and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+                me.
+            </p>
+            <p class="headline-sm-suport">LET’S CONNECT</p>
+            <div class="social-icons">
+                <i class="fa-brands fa-twitter"></i>
+                <i class="fa-brands fa-facebook"></i>
+                <i class="fa-brands fa-linkedin"></i>
+                <i class="fa-brands fa-medium"></i>
+                <i class="fa-brands fa-github"></i>
+            </div>
```

Comment:
- [OPTIONAL] KIndly make sure all social media links are wrapped into the `ul` tag and make sure every link is rendered using the Anchor tag and wrapped into the `li` tag. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href` attribute.